scripts/zones/Oldton_Movalpolos/mobs/Goblin_Wolfman.lua

```diff
@@ -0,0 +1,39 @@
+-----------------------------------
+-- Area: Oldton Movalpolos (11)
+--  Mob: Goblin Wolfman
+-----------------------------------
+
+function onMobDeath(mob, player, isKiller)
+end;
+
+
+function onMobSpawn(mob)
+    local GoblinWolfmanID = mob:getID();
+
+   -- Set the hp% to use blood weapon
+    mob:setLocalVar("jobSpecTrigger", math.random(50, 75));
```

Comment:
The mixin defaults to 40-60%.  If you want a range different from default, you'd set it like this in onMobSpawn().
```
    mob:setLocalVar("useSpecAtHpMin", 50);
    mob:setLocalVar("useSpecAtHpMax", 75);
```